Task_DEV_6/Task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace Task_DEV_6
+{
+    class CustomStack
+    {
+        private const int LENGTH = 10;
+        private const string EXCEPTION_MESSAGE = "There are no elements on the stack.";
+
+        private int[] elements;
+        private int count;
+
+        public CustomStack()
+        {
+            elements = new int[LENGTH];
+        }
+
+        public CustomStack(int capacity)
+        {
+            elements = new int[capacity];
+        }
+
+        public int Capacity()
+        {
+            return elements.Length;
+        }
+
+        public int Count()
+        {
+            return count;
+        }
+
+        public bool isEmpty()
+        {
+            if (count == 0)
+            {
+                return true;
+            }
+            return false;
+        }
+
+        public int Pop()
+        {
+            ThrowExeption();
```

Comment:
Why do we throw exception at the beginning of the method??? Without some checks.

Reply:
Done.